src/openrct2/actions/WaterSetHeightAction.cpp

```diff
@@ -21,6 +21,12 @@ WaterSetHeightAction::WaterSetHeightAction(const CoordsXY& coords, uint8_t heigh
 {
 }
 
+void WaterSetHeightAction::AcceptParameters(GameActionParameterVisitor& visitor)
```

Comment:
As above do we need both the raise/lower and the set height actions exposed to plugins.